tsconfig.json

```diff
@@ -0,0 +1,78 @@
+{
+	"exclude": [
+		"./dist/"
+	],
+	"include": [
+		"./src/**/*.ts"
+	],
+	"compilerOptions": {
+		"module": "commonjs",
+		"target": "esnext",
+		"noImplicitAny": true,
+		"sourceMap": false,
+		"emitDecoratorMetadata": true,
+		"experimentalDecorators": true,
+		"strictNullChecks": false,
+		"noImplicitThis": true,
+		"rootDir": "./src/",
+		"rootDirs": [
+			"./src/",
+			"./tests/"
+		],
+		"allowJs": false,
+		"allowUnreachableCode": false,
+		"allowUnusedLabels": false,
+		"alwaysStrict": true,
+		"baseUrl": "",
+		"charset": "utf8",
+		"declaration": true,
+		// "declarationDir": "./dts/",
```

Comment:
Please drop commented lines from this JSON file. Is this conventional for tsconfig?